Misc/NEWS.d/next/Library/2021-01-03-13-44-44.bpo-42815.IJqHPr.rst

```diff
@@ -0,0 +1,2 @@
+Fix issue when new thread doesn't copy context of a parent thread. Patch
```

Comment:
This description is not very helpful. I would prefer to rephrase it to explain that threads no longer **share** contextvars variables with their parent thread, but the context is now **copied**.

IMO this change is important enough to be documented in a ``.. versionchanged:: 3.11`` markup in https://docs.python.org/dev/library/threading.html#threading.Thread documentation.

Reply:
Actually every thread has it's own context and they do not share same context. This PR tries fix problem when child thread use empty context instead of copy of a parent context.

Reply:
Please rephrase the NEWS entry and add the requested versionchanged markup.